lib/trust-services-manager.js

```diff
@@ -227,29 +228,114 @@ class TrustServicesManager extends PkiExpressOperator {
 		});
 	}
 
+	/** 
+	 * Starts the authentication with all configured trusted service
+	 *
+	 * @param {string} redirectUrl
+	 *
+	 *
+	 * @param {SessionType} sessionType @see TrustServiceSessionTypes
+	 *
+	 * @param {string} sessionLifetime
+	 *
+	 * @param {string} customState
+	 *
+	 * @returns {StartServiceAuthResult}
+	 */
+	startAuth(
+		redirectUrl,
+		sessionType = TrustServiceSessionTypes.SIGNATURE_SESSION,
+		sessionLifetime = null,
+		customState = null
+	) {
+		if (!redirectUrl) {
+			throw new Error('No redirectUrl was provided');
+		}
+
+		let args = [
+			redirectUrl,
+		];
+
+		if (sessionType) {
+			args.push('--session-type');
+			args.push(sessionType);
+		}
+		if (sessionLifetime) {
```

Comment:
Although the command receives this parameter `sessionLifetime` as a `string`, I think it's more clear to receive a `integer` here:

1. Create a validation in the beginning of the method:
```
if (!isNaN(sessionLifetime)) {
   throw new Error('The argument "sessionLifetime" should be a number (representing seconds)'):
}
```
2. Convert this number into a string when adding the argument:
```
if (sessionLifetime) {
    args.push('--session-lifetime'):
    args.push(sessionLifetime.toString());
}
```